open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/admin/OpenLineageServerOperationalServices.java

```diff
@@ -136,9 +136,8 @@ private void initializeOLS(OpenLineageServerConfig openLineageServerConfig) thro
         final String methodName = "initializeOLS";
         final String actionDescription = "Initialize Open lineage Services";
         Connection lineageGraphConnection = openLineageServerConfig.getLineageGraphConnection();
-        OLSSimplifiedAccessServiceConfig accessServiceConfig = openLineageServerConfig.getAccessServiceConfig();
 
-        Connection inTopicConnection = getAssetLineageOutTopicConnection(methodName, accessServiceConfig);
+        Connection inTopicConnection = getAssetLineageOutTopicConnection(openLineageServerConfig, methodName);
```

Comment:
At this point before the connector is create we need to merge the properties if they are defined in "assetLineageTopicConnectionOverride" coming form the server configuration document. 
Based on the current code, it looks like we need to replace configurationProperties map for the first object in the embeddedConnections we retrieved (the object inTopicConnection above).

=)